### ISSUE TITLE
Product details

### ISSUE THREAD

**Issue by tmathern** [hlxsites/sgws-creative#101]
Section displayed in horizontal accordion (the section being the fragment) with multiple variations.

Variation 1: 1 product image (right) and text on the left
<img width="1415" alt="Screenshot 2023-06-23 at 11 10 29 AM" src="https://github.com/hlxsites/sgws-creative/assets/60901087/8c3e3829-10e7-4a6d-a33c-dab88494ad62">

Variation 2: Two product images with text left and right
<img width="1415" alt="Screenshot 2023-06-23 at 11 11 02 AM" src="https://github.com/hlxsites/sgws-creative/assets/60901087/923cbd51-ae2b-4621-b1a1-be73ff35dc59">

Variation 3: Multiple product images with single-line text on top
<img width="1415" alt="Screenshot 2023-06-23 at 11 11 36 AM" src="https://github.com/hlxsites/sgws-creative/assets/60901087/7d4c7ce7-065f-49b6-bcc8-354f01ca5e5a">

Variation 4: 2 product images with stats
<img width="1415" alt="Screenshot 2023-06-23 at 11 10 43 AM" src="https://github.com/hlxsites/sgws-creative/assets/60901087/02b548f2-7fd9-491b-aeac-dd7c0980c59a">

Note how all have the same bottom background, and have some transparency to see the video behind hit.

Original Page: https://sgcreative.southernglazers.com/bloominbrands/p/7

**Comment by badvision** [comment]
Current progress is in the product-details branch [1]. I have the wood table style implemented, you just need to apply "wood table" as a variant to the section of the fragment for it to be picked up and applied correctly.  This allows us to have tabs which also do not get the wood table as well.

This branch also has a few fixes for the charts (though I realized after the fact this was not in this story per se) so that the charts take a more appropriate height to fill the container div that they are in if that container is a vertical tab.

Finally, variant #3 was implemented and required a lot of media breakpoints to resize the content area using a transform scale approach.  I tried other ways such as calc-based approaches but at best it was difficult to get the images to stay proportional to each other as they became smaller.

Variants 1 and 2 are probably not as difficult, and we might really want to punt variant 4 to its own story for someone to work on in parallel -- it will not be trivial to do.

[1] https://github.com/hlxsites/sgws-creative/tree/product-details
